packages/js/src/utils/errors.ts

```diff
@@ -32,12 +32,19 @@ const getMessageByErrorCode = (errorCode: LogtoErrorCode): string => {
   return errorCode;
 };
 
+interface LogtoErrorParameters {
+  code: LogtoErrorCode;
+  error?: string;
+  errorDescription?: string;
+  message?: string;
+}
+
 export class LogtoError extends Error {
   code: LogtoErrorCode;
   error?: string;
   errorDescription?: string;
 
-  constructor(code: LogtoErrorCode, error?: string, errorDescription?: string, message?: string) {
+  constructor({ code, error, errorDescription, message }: LogtoErrorParameters) {
```

Comment:
Most of logtoErrors only need one arg: `code`, how about this:
```
constructor(payload: LogtoErrorParameters | string)
```

Reply:
Good idea.

Reply:
Updated, PTAL.
```
constructor(payload: LogtoErrorCode | LogtoErrorParameters)
```